lib/assesments/assessment_2/amna/screens/profile/widget/fi_as2_amna_stats.dart

```diff
@@ -0,0 +1,54 @@
+part of '../fi_as2_amna_profile.dart';
+
+class _FiAs2AmnaStats extends StatelessWidget {
+  const _FiAs2AmnaStats({super.key});
+  static const List<String> titles = ["Photos", "Followers", "Follows"];
+  static const List<String> numbers = ["315", "77.5k", "500"];
+  @override
+  Widget build(BuildContext context) {
+    const smallWidth = SizedBox(
+      width: 50,
+    );
+    return Padding(
+      padding: const EdgeInsets.only(right: 16, left: 16),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: buildStatBar(smallWidth),
+      ),
+    );
+  }
+
+  List<Widget> buildStatBar(Widget separator) {
```

Comment:
instead of segregating widgets as functions make classes, functions cloud stack when build gets called again and again, making the app slow, classes do not have this problem,